src/Resources/scaffolds/6.0/change-password/src/Controller/ChangePasswordController.php.tpl

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\User;
+use App\Form\ChangePasswordFormType;
+use App\Repository\UserRepository;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\PasswordHasher\Hasher\UserPasswordHasherInterface;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Security\Core\User\UserInterface;
+
+#[Route('/user/change-password', name: 'change_password')]
+class ChangePasswordController extends AbstractController
+{
+    public function __invoke(
+        Request $request,
+        UserPasswordHasherInterface $userPasswordHasher,
+        UserRepository $userRepository,
+        ?UserInterface $user = null,
```

Comment:
Hmm not sure, the better way would be to use the `CurrentUser` parameter attribute to inject the actual entity. I believe this causes problems when using the framework-extra-bundle (it would try and load from db).